src/Pods/Admin/Config/Pod.php

```diff
@@ -735,11 +736,12 @@ public function get_fields( \Pods\Whatsit\Pod $pod, array $tabs ) {
 					'boolean_yes_label' => '',
 				],
 				'rewrite_custom_slug'    => [
-					'label'      => __( 'Custom Rewrite Slug', 'pods' ),
-					'help'       => __( 'Changes the first segment of the URL, which by default is the name of the Pod. For example, if your Pod is called "foo", if this field is left blank, your link will be "example.com/foo/post_slug", but if you were to enter "bar" your link will be "example.com/bar/post_slug".', 'pods' ),
-					'type'       => 'text',
-					'default'    => '',
-					'depends-on' => [ 'rewrite' => true ],
+					'label'         => __( 'Custom Rewrite Slug', 'pods' ),
+					'help'          => __( 'Changes the first segment of the URL, which by default is the name of the Pod. For example, if your Pod is called "foo", if this field is left blank, your link will be "example.com/foo/post_slug", but if you were to enter "bar" your link will be "example.com/bar/post_slug".', 'pods' ),
+					'type'          => 'slug',
+					'slug_fallback' => '-',
+					'default'       => '',
+					'depends-on'    => [ 'rewrite' => true ],
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: b0f99a1faba54e57496d114d833d9661</sub>